scripts/signo-chino.js

```diff
@@ -0,0 +1,84 @@
+// Description:
+//   Devuelve el signo Chino dependiendo del año
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   hubot signo chino 1989
+//
+// Author:
+//   @raerpo
+
+module.exports = robot => {
+  robot.respond(/signo chino(.*)/i, function(msg) {
```

Comment:
@raerpo no es mejor cambiar el `pattern` a `signo chino (\d{4})` y así evitar validar que no venga vacío?

Reply:
Si lo agrego asi como propones, el problema que veo es que  cuando la gente escriba:
`huemul signo chino`
No va a pasar nada. En cambio de esta manera el :huemul: les va a decir que les falta el año.

Tome el enfoque de que el script se ejecute usando el `signo chino` y dar el feedback de lo que se debe corregir en vez de fallar silenciosamente evitando correr el script.

Cual deberia ser la forma correcta?

Reply:
UH pelea!

Reply:
para eso esta el comando `help signo chino`. Ese comando te entrega el como usar el script

Reply:
yo apoyo lo que hizo @raerpo because user is weón first.

Reply:
creo que ese enfoque solo hace escribir mas código y al final no se promueve el uso de `help`. Dejando eso de lado (:nail_care:) el resto TAWENO

Reply:
no todos tienen `help` ese es el tema.

Reply:
este caso si tiene help